src/m365/spo/commands/web/web-roleassignment-remove.ts

```diff
@@ -114,15 +131,29 @@ class SpoWebRoleAssignmentRemoveCommand extends SpoCommand {
     try {
       if (options.upn) {
         options.principalId = await this.getUserPrincipalId(options);
-        await this.removeRoleAssignmentWithOptions(logger, options);
       }
       else if (options.groupName) {
         options.principalId = await this.getGroupPrincipalId(options);
-        await this.removeRoleAssignmentWithOptions(logger, options);
       }
-      else {
-        await this.removeRoleAssignmentWithOptions(logger, options);
+      else if (options.entraGroupId || options.entraGroupName) {
+        if (this.verbose) {
+          await logger.logToStderr('Retrieving group information...');
+        }
+
+        let group: Group;
+        if (options.entraGroupId) {
+          group = await entraGroup.getGroupById(options.entraGroupId);
+        }
+        else {
+          group = await entraGroup.getGroupByDisplayName(options.entraGroupName!);
+        }
```

Comment:
this may be simplified to
```suggestion
          const group = options.entraGroupId
            ? await entraGroup.getGroupById(options.entraGroupId)
            : await entraGroup.getGroupByDisplayName(options.entraGroupName!);
```